### ISSUE TITLE
(Mac) Redner crashing at 7 frames rendered consistently

### ISSUE THREAD

**Issue by graemc** [ReplayMod/ReplayMod#215]
I'm on Minecraft 1.12.2 and every time I try to render my clip, it crashes at 7 frames rendered. coincidentally. i've tried different places for the file, different resolutions, taking out every other mod in my mods folder, jumping to the newest version, (2.3.1) Help? if i've missed any information, ask me because im sure I've missed something. 
[latest.log](https://github.com/ReplayMod/ReplayMod/files/4748089/latest.log)

**Comment by Johni0702** [comment]
Does it work if you keep the default output file?
If it doesn't, please attach the log file for that.

**Comment by graemc** [comment]
> Does it work if you keep the default output file?
> If it doesn't, please attach the log file for that.

no, i've tried the default output file location and on the desktop

**Comment by graemc** [comment]
> > Does it work if you keep the default output file?
> > If it doesn't, please attach the log file for that.
> 
> no, i've tried the default output file location and on the desktop

Oh, sry. I was supposed to put in the log, I’ll do that tmr as I’m going to bed

**Comment by graemc** [comment]
> > > Does it work if you keep the default output file?
> > > If it doesn't, please attach the log file for that.
> > 
> > 
> > no, i've tried the default output file location and on the desktop
> 
> Oh, sry. I was supposed to put in the log, I’ll do that tmr as I’m going to bed

ok here is a video and the log:
[latest.log](https://github.com/ReplayMod/ReplayMod/files/4755297/latest.log)
[Bully Game.zip](https://github.com/ReplayMod/ReplayMod/files/4755300/Bully.Game.zip)

**Comment by Johni0702** [comment]
Hm, the log still says:
> "Abandoned Castle Build.mp4"

**Comment by graemc** [comment]
> Hm, the log still says:
> 
> > "Abandoned Castle Build.mp4"

yeah, it also says: 

[15:53:29] [main/WARN] [FML]: The coremod com.replaymod.core.LoadingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
[15:53:29] [main/WARN] [FML]: The coremod LoadingPlugin (com.replaymod.core.LoadingPlugin) is not signed!

could that be the reason?

**Comment by Johni0702** [comment]
> > Hm, the log still says:
> > > "Abandoned Castle Build.mp4"
> 
> yeah, it also says:
> 
> [15:53:29] [main/WARN] [FML]: The coremod com.replaymod.core.LoadingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
> [15:53:29] [main/WARN] [FML]: The coremod LoadingPlugin (com.replaymod.core.LoadingPlugin) is not signed!
> 
> could that be the reason?

No, that's merely a warning and unless you're using the wrong version of RM for your MC version (in which cause it'd probably already blow up during loading), it's not an issue.

**Comment by graemc** [comment]
> > > Hm, the log still says:
> > > > "Abandoned Castle Build.mp4"
> > 
> > 
> > yeah, it also says:
> > [15:53:29] [main/WARN] [FML]: The coremod com.replaymod.core.LoadingPlugin does not have a MCVersion annotation, it may cause issues with this version of Minecraft
> > [15:53:29] [main/WARN] [FML]: The coremod LoadingPlugin (com.replaymod.core.LoadingPlugin) is not signed!
> > could that be the reason?
> 
> No, that's merely a warning and unless you're using the wrong version of RM for your MC version (in which cause it'd probably already blow up during loading), it's not an issue.

oh ok, do you have an idea on what could be the issue then? because if thats not it, i'm blank. :|

**Comment by Johni0702** [comment]
> oh ok, do you have an idea on what could be the issue then? because if thats not it, i'm blank. :|

Yes and I already told you: https://github.com/ReplayMod/ReplayMod/issues/215#issuecomment-640836466
